app/main.py

```diff
@@ -1 +1,19 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    comand, source_file, destination = command.split()
```

Comment:
you should check length of command before unpacking in order to avoid unexpected error